.pipelines/stages/jobs/steps/capi-win-step.yml

```diff
@@ -31,6 +34,13 @@ steps:
     echo "cuda_version=$(cuda_version)"
     echo "target=${{ parameters.target }}"
   displayName: 'Print Parameters'
+
+#- template: utils/download-ort-build.yml
```

Comment:
Why this part is commented out? Is it still needed or not?

Reply:
It is not, I wil trying to use the pipeline build artifacts instead of ado feed